SR2020/2020-05-26-virtual-competition-details.md

```diff
@@ -0,0 +1,32 @@
+---
+to: sr2020-teams
+subject: Student Robotics 2020 Virtual Competition Details
+---
+
+![](https://pbs.twimg.com/media/EYt9wgcX0AAKjOn?format=jpg)
+
+Hello!
+
+Today's the day we announce some long awaited details of the virtual competition for Student Robotics 2020.
+
+## Simulator
+
+For the virtual competition, we've created a new simulation, based in [webots](https://cyberbotics.com/). Documentation on how to download and get started with the simulator is available in the docs.
+
+[Competition Simulator Docs](https://studentrobotics.org/docs/competition-simulator/)
+
+Due to some compatibility and API changes, it's unlikely your existing code will work exactly in the simulator, however we've tried to keep it as similar as possible, and easily adaptable from what you've already written.
```

Comment:
We should probably clarify that the online IDE is not going to work well for developing for the virtual competition. Not sure if that needs to be in this email, but we do need to mention it somewhere (and obviously so). This paragraph is talking about the changes, so feels like it might fit.
Regardless of whether it's mentioned here, we definitely do need to mention it in the docs.

Reply:
I'm going to er towards the docs, as they're more likely to be read directly by students